### ISSUE TITLE
fabric.mod.json isn't generated by template 

### ISSUE THREAD

**Issue by QazCetelic** [DemonWav/MinecraftDevIntelliJ#1831]
`fabric.mod.json` isn't generated by template, which means that the mod won't be initialized. It won't cause any error, but does mean that the generated build won't do anything and none of the code is used with the `runClient` Gradle task.

Please include the following information in all bug reports:

* Minecraft Development for IntelliJ plugin version: 2022.1-1.5.19
* IntelliJ version: IntelliJ IDEA 2022.1 (Community Edition)
* Operating System: Linux (Fedora)
* Target platform: Fabric

**Comment by SzczurekYT** [comment]
Hello, this happened to me today and yesterday.

**Comment by SzczurekYT** [comment]
Any updates on this?

**Comment by SzczurekYT** [comment]
Hmm.
I created new project and Intellij told me it removed a module. I restored it (there was a button to do that) and this time it worked.

**Comment by SzczurekYT** [comment]
@DenWav  any help? Sorry for a ping but this plugin is so usefull and now I can't make fabric mods easily.

**Comment by DenWav** [comment]
I thought this was fixed. I'll try to remember to take a look

**Comment by SzczurekYT** [comment]
> I thought this was fixed. I'll try to remember to take a look

Thanks

**Comment by SzczurekYT** [comment]
> I thought this was fixed. I'll try to remember to take a look

Overall it works randomly, for example right now I created new project, on latest Intellij and latest nightly build and got no fabric.mod.json and no client initializer. Sometimes It doesn't generates normal, both side initializer if I remember correctly.

My plugin version: 2022.1-1.5.20-333
Intellij version: 2022.1.4
OS: Windows 10
Edit: The project was generated for 1.18.2.

**Comment by SzczurekYT** [comment]
I'm not sure but there is a chance that #765 is related, I think this also happend to me one time.

**Comment by SzczurekYT** [comment]
That's weired, today I generated new project and this time it worked completly fine, with mixin package being the only missing thing.

**Comment by SzczurekYT** [comment]
Generated another one, this time all the files under src directory are missing. 
![image](https://user-images.githubusercontent.com/67533827/183239258-69a89fcb-d4bc-423c-83c9-361885593cc8.png)

**Comment by SzczurekYT** [comment]
I can't confirm this, but I think the issue is caused by unticking the use fabric api checkbox.

**Comment by SzczurekYT** [comment]
> I can't confirm this, but I think the issue is caused by unticking the use fabric api checkbox.

I think that's wrong, that was not the case.

Will this get fixed?
Few days ago I got a fresh linux installation with dual boot setup (my windows 10 is still there). I installed latest version of intellij and minecraft dev and the bug is still alive!!!
This is just so annoying that you get **almost** complete project.
Still the fabric.mod.json is not generated.
Other files were fine.
Or am I doing something wrong?
There is this load gradle project popup. Should I click it when It shows up or just ignore and wait for the project to finish generating?
I did a test now and it seems that not clicking this works better.
When I clicked both `java` and `resources` directories were empty but when I just waited the `fabric.mod.json` was the only missing file.
